src/Tribe/Tickets_Handler.php

```diff
@@ -1053,23 +1062,37 @@ public function save_tickets_order( $post_id ) {
 			return;
 		}
 
-		if ( ! empty( $_POST['tribe_tickets_order'] ) ) {
-			$ticket_order = str_ireplace ( 'order_', '', $_POST['tribe_tickets_order'] );
-			$ticket_order = explode( ',', $ticket_order );
-			$ticket_order = array_flip( $ticket_order );
+		$this->saving_order = true;
 
-			foreach ( $ticket_order as $id => $order ) {
-				wp_update_post( array(
-					'ID'           => absint( $id ),
-					'menu_order'   => absint( $order ),
-				) );
-			}
+		$ticket_order = $_POST['tribe_tickets_order'];
+
+		update_post_meta(
+			$post_id,
+			$this->tickets_order_field,
+			$ticket_order
+		);
+
+		$ticket_order = explode( ',', $ticket_order );
+		$ticket_order = array_flip( $ticket_order );
+
+		foreach ( $ticket_order as $id => $order ) {
+			wp_update_post( array(
+				'ID'           => absint( $id ),
+				'menu_order'   => absint( $order ),
```

Comment:
there are two spaces between `'menu_order'` and  `=>` here — reduce to one space, and align the spacing of `'ID'` and _its_ `=>` accordingly